_codux/boards/card-collection-1.board.tsx

```diff
@@ -0,0 +1,70 @@
+import './card-collection-1.board.css';
+import { createBoard } from '@wixc3/react-board';
+
+export default createBoard({
+    name: 'Card Collection 1',
+    Board: () => (
+        <div className="card-collection">
+            <h1 className="card-collection-title">
+                Write a Title Here.
+                <br /> Click to Edit and Add Your Own Text.
+            </h1>
+            <div className="card-collection-items">
+                <div className="card">
+                    <div className="card-image-box">
+                        <img
+                            src="https://static.wixstatic.com/media/8586f3_f6d2538ac6374f71b6dc29511c5cd1a3~mv2.png/v1/fill/w_860,h_600/Frame%206.png"
+                            alt="Delicious Margherita Pizza"
```

Comment:
I think that having an alt value is a benefit as we insentivise the users to set them, maybe we want to use a different value than this.